Makefile

```diff
@@ -81,3 +85,18 @@ pip-requirements-test: venv
 		pip install -e ".[test]" > /dev/null
 
 .PHONY: dist
+else
+PYTHON_VERSION = 3.9
+PWD = $(shell pwd -P)
```

Comment:
This is overriding an existing default variable, that generally contains the path PWD where `make` was called. It seems what you want here is the directory where the Makefile is working, that is `$(CURDIR)`, which seems to already be resolved to a physical path (rather than one with symlinks):

```
~/work$ cat Make-test/Makefile
test:
        @echo PWD=$(PWD)
        @echo CURDIR=$(CURDIR)
        @echo pwd -P=$(shell pwd -P)

~/work$ make -C Make-test
PWD=/Users/shtrom/work
CURDIR=/Volumes/code/Make-test
pwd -P=/Volumes/code/Make-test
```